deploy/demo/aadpodidentity.yaml

```diff
@@ -4,5 +4,5 @@ metadata:
  name: demo-aad1
 spec:
  type: 0
- ResourceID: RESOURCE_ID
- ClientID: CLIENT_ID 
+ resourceID: RESOURCE_ID
```

Comment:
We should update these manifests right before we cut the release as this will not work with current release.